### PR TITLE
complete kata 7 post new comment

### PR DESCRIPTION
**Pull request by imaa04**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -156,4 +156,55 @@ describe("/api/articles/:article_id/comments", () => {
...
+      body: "This article is lit!",
+    };
+    return request(app)
+      .post("/api/articles/bad_id/comments")
+      .send(newComment)
+      .expect(400)
+      .then((response) => {
+        expect(response.body.msg).toBe("Bad request");
+      });
+  });
```
Comment (Keb84) [inline __tests__/app.test.js:209]: There are couple more tests needed here - 
Status 400 - missing required fields for example `body : "test"`
Status 404 - username doesn't exist in the database
Status 201 - ignores unnecessary fields

**Review by Keb84 — CHANGES_REQUESTED** [review]
Really nice work 🚀 please see my comments, add the tests and resubmit 👍

File: controllers/articles.controllers.js [hunk 2]
```diff
@@ -41,3 +42,22 @@ exports.getCommentByArticleId = (req, res, next) => {
     })
     .catch(next);
 };
+
+exports.postNewComment = (req, res, next) => {
+    const {article_id} =req.params
+    const newComment = req.body
+    const commentPromises = [checkArticleExists(article_id)];
```
Comment (nickkagame) [inline controllers/articles.controllers.js:49]: check to see article exists isn't really necessary.  You can just throw a custom error. An empty array coming back in rows in your model will mean that article doesn't exist. No need to change though

**Review by nickkagame — APPROVED** [review]
Looking good Ima!  Minor comments only.

**Pull request merged**